front/src/reducers/infra/index.ts

```diff
@@ -0,0 +1,44 @@
+import { createSlice } from '@reduxjs/toolkit';
+import type { CaseReducer, Draft, PayloadAction } from '@reduxjs/toolkit';
+import type { Feature } from 'geojson';
+
+export interface InfraStateType {
```

Comment:
No need to add Type at the end, the capital letter already indicates that it is a type
```suggestion
export interface InfraState {
```